docker/fedora/keylime_rust.Dockerfile

```diff
@@ -47,10 +48,16 @@ tpm2-abrmd \
 tpm2-tools \
 tpm2-tss \
 tpm2-tss-devel \
-uthash-devel"
+uthash-devel \
+czmq-devel"
 
 RUN dnf makecache && \
   dnf -y install $PKGS_DEPS && \
   dnf clean all && \
   rm -rf /var/cache/dnf/*
 
+# Move keylime.conf to expected location in /etc/
+WORKDIR ${KEYLIME_HOME}
+RUN git clone https://github.com/keylime/keylime.git $KEYLIME_HOME
```

Comment:
nit: consider combining the RUN with `&& \` so it's one layer.

Reply:
Good point, thanks.